src/pages/createPost.tsx

```diff
@@ -47,229 +69,236 @@ const createPost = () => {
-      title: 'Coffee',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1533827432537-70133748f5c8',
-      title: 'Hats',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1558642452-9d2a7deb7f62',
-      title: 'Honey',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1516802273409-68526ee1bdd6',
-      title: 'Basketball',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1518756131217-31eb79b20e8f',
-      title: 'Fern',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1597645587822-e99fa5d45d25',
-      title: 'Mushrooms',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1567306301408-9b74779a11af',
-      title: 'Tomato basil',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1471357674240-e1a485acb3e1',
-      title: 'Sea star',
-    },
-    {
-      img: 'https://images.unsplash.com/photo-1589118949245-7d38baf380d6',
-      title: 'Bike',
-    },
-  ];
 
-    const [value, setValue] = React.useState<Dayjs | null>(
-      dayjs('2014-08-18T21:11:54'),
-    );
-  
-    const handleChange = (newValue: Dayjs | null) => {
-      setValue(newValue);
-    };
-
-  
-  const Location = [
-    {title:"Poseidon entertainment complex"},
-    {title:"Jod Fair"}
-  ]
-  const Tags= [
-    {Tag:"RPG"},
-    {Tag:"Car"},
-    {Tag:"CO-OP"},
-  ]
-  const ListItem = styled('li')(({ theme }) => ({
-    margin: theme.spacing(0.5),
-  }));
-  const [chipData, setChipData] = React.useState<readonly ChipData[]>([
-    { key: 0, label: 'Angular' },
-    { key: 1, label: 'jQuery' },
-    { key: 2, label: 'Polymer' },
-    { key: 3, label: 'React' },
-    { key: 4, label: 'Vue.js' }
-  ]);
+  const [startDate, setStartDate] = React.useState<Dayjs | null>(
+    dayjs(initialValue.startDateTime === "" ? null : initialValue.startDateTime)
+  );
+  const [endDate, setEndDate] = React.useState<Dayjs | null>(
+    initialValue.endDateTime === "" ? null : dayjs(initialValue.endDateTime)
+  );
+  const [selectedTags, setSelectedTags] = useState(initialValue.tags);
+  const [images, setImages] = useState<string[]>([]);
+  useEffect(() => {
+    setImages(dummyEditFrom.image);
+  }, [dummyEditFrom.image]);
 
-  const handleDelete = (chipToDelete: ChipData) => () => {
-    setChipData((chips) => chips.filter((chip) => chip.key !== chipToDelete.key));
+  const handleAddTag = (event: any, tag: any) => {
+    if (tag.length <= 5) {
+      console.log(tag.length);
+      setSelectedTags(tag);
+    }
   };
 
-  const handleAddTag = (chipToAdd: ChipData) => () => {
-    //setChipData((chips) => chips.filter((chip) => chip.key !== chipToDelete.key));
-    //I don't know how to do, But When click it i want it to show selector(list of tag) then i can click it to add into chipData
+  const handleStartDate = (newValue: Dayjs | null) => {
+    if (newValue && newValue.isBefore(dayjs())) {
+      console.error("Start date must be in the future.");
```

Comment:
I think since user may not look at the console, you have to make the error appears on web not in console.